x-pack/plugins/security_solution/public/management/components/console/components/command_input/key_capture.tsx

```diff
@@ -23,9 +23,11 @@ const NOOP = () => undefined;
 
 const KeyCaptureContainer = styled.span`
   display: inline-block;
-  position: relative;
+  position: absolute;
```

Comment:
Why did you need to this?

Reply:
I needed to remove the element from the normal flow of the HTML document so that it would not take up space in the UI. don't remember now why this was not an issue in the prior implementation, but setting it to `absolute` addresses has no impact on this component or its functionality (its a hidden component)